src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -791,31 +797,33 @@ void StateControllerImpl::OnApplicationRegistered(
   OnStateChanged(app, initial_state, new_state);
 }
 
-int64_t StateControllerImpl::SendBCActivateApp(
+int64_t StateControllerImpl::SendStateToHMI(
     ApplicationConstSharedPtr app,
     hmi_apis::Common_HMILevel::eType level,
     bool send_policy_priority) {
   LOG4CXX_AUTO_TRACE(logger_);
-  smart_objects::SmartObjectSPtr bc_activate_app_request =
-      MessageHelper::GetBCActivateAppRequestToHMI(
-          app,
-          app_mngr_.connection_handler().get_session_observer(),
-          app_mngr_.GetPolicyHandler(),
-          level,
-          send_policy_priority,
-          app_mngr_);
-  if (!bc_activate_app_request) {
-    LOG4CXX_ERROR(logger_, "Unable to create BC.ActivateAppRequest");
+  smart_objects::SmartObjectSPtr request = NULL;
+  if (level == hmi_apis::Common_HMILevel::NONE) {
```

Comment:
@jacobkeeler good practice to use the variable to the right of comparison

Reply:
@LuxoftAKutsan fixed with https://github.com/smartdevicelink/sdl_core/pull/2948/commits/8080f10bd7b7f5bbe074cbdafdc19fba7ebe8118